src/Dev/TestKernel.php

```diff
@@ -41,6 +46,21 @@ protected function getIncludeTests()
         return true;
     }
 
+
+    /**
+     * @param string[] $ciLibs
+     */
+    public function setIgnoreCILibraries(array $ciLibs): self
```

Comment:
I didn't want to create yet another TestKernel subclass, so I figured adding a setter/getter was a cleaner option.

Reply:
There was discussion here - https://github.com/silverstripe/silverstripe-framework/pull/10016#discussion_r747163402 - about some refactoring the Kernel and that's been implemented on the PR - it may be applicable for what you're doing here

Reply:
I think `CoreKernel`, `TestKernel`and @unclecheese's new `DatabaselessKernel` are all trying to do substantially different things. So it makes sense to keep all of them as separate classes.

The differences between `TestKernel` with PHPUnit5 and `TestKernel` with PHPUnit9 are a lot more subtle. It's fundamentally a variation on the same use case.

But it's good to point out that these two PR will probably have a merge conflict at some stage.